win32/gui.py

```diff
@@ -845,6 +845,10 @@ def kill_old_process():
     if Platform.is_windows():
         # Let's kill any other running instance of our GUI/SystemTray before starting a new one.
         kill_old_process()
+        if len(sys.argv) > 1 and "-stop" in sys.argv:
+            # just return.  The kill_old_process() should have terminated the process,
+            # and now we're done.
+            sys.exit(0)
```

Comment:
I can take it out if need be. Originally, I thought I was going to need to "shell out" from the installer (in Windows parlance, a Custom Action) to stop the agent-manager if it's running.  Turns out I didn't need to.  However, it seems potentially useful to be able to stop it from the command line.  There'd be no harm removing.